Userland/Libraries/LibWeb/CSS/CSS.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021-2023, Sam Atkins <atkinssj@serenityos.org>
+ * Copyright (c) 2023, Tim Flynn <trflynn89@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Runtime/VM.h>
+#include <LibWeb/CSS/CSS.h>
+#include <LibWeb/CSS/Parser/Parser.h>
+#include <LibWeb/CSS/PropertyID.h>
+#include <LibWeb/CSS/Serialize.h>
+
+namespace Web::CSS {
+
+// https://www.w3.org/TR/cssom-1/#dom-css-escape
+WebIDL::ExceptionOr<String> escape(JS::VM& vm, StringView identifier)
+{
+    return TRY_OR_THROW_OOM(vm, serialize_an_identifier(identifier));
```

Comment:
`// The escape(ident) operation must return the result of invoking serialize an identifier of ident.`